recipes/codacy-coverage/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "codacy-coverage" %}
+{% set version = "1.3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b94651934745c638a980ad8d67494077e60f71e19e29aad1c275b66e0a070cbc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - requests >=2.9.1
+  run:
+    - python
+    - requests >=2.9.1
+
+test:
+  commands:
+    - python-codacy-coverage --help
+
+about:
+  home: https://github.com/codacy/python-codacy-coverage
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

You don't need to use the `RECIPE_DIR` variable here. By default it looks in the work directory (where the source is extracted) before falling back to the recipe directory.
```suggestion
  license_file: LICENSE
```